tdms/include/arrays.h

```diff
@@ -172,6 +196,16 @@ class DispersiveMultiLayer{
 
 public:
   explicit DispersiveMultiLayer(const mxArray *ptr);
+
+  /**
+   * @brief Determines whether the (background) medium is dispersive
+   *
+   * @param K_tot Number of Yee cells in the z-direction (number of entries in this->gamma)
+   * @param near_zero_tolerance Tolerance for non-zero gamma (attenuation) values
+   * @return true Background is dispersive
+   * @return false Background is not dispersive
+   */
+  bool is_dispersive(int K_tot, double near_zero_tolerance=1e-15);
```

Comment:
So we can have a non-dispersive dispersive multilayer? (`DispersiveMultiLayer::is_dispersive == false`)